code-conventions.md

```diff
@@ -1,36 +1,43 @@
-# Coding Conventions
+# Code Conventions
 
 ## Security
-(this is very important)
+Keep in mind that your code will be running on servers; security matters.
```

Comment:
This sentence doesn't really make sense without a quantity, should be something like "your code will be running on **hundreds** of servers"

Reply:
Thousands, even!

![image](https://user-images.githubusercontent.com/12614053/75294863-bf068380-5829-11ea-8ab5-749ba4874d58.png)

Reply:
I thought about mentioning this, but decided to leave it out. I want people to realize that the code they're writing will be used to service untrusted clients, not to brag with bStats popularity. Then again, maybe it would drive the point home better.